dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -89,22 +90,15 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
         THROW_IF_FAILED(notificationPlatform->RegisterFullTrustApplication(processName.get(), remoteId, &unpackagedAppUserModelId));
     }
 
-    winrt::hresult CreateChannelWithRemoteIdHelper(const winrt::guid& remoteId, ChannelDetails& channelInfo) noexcept try
+    winrt::hresult CreateChannelWithRemoteIdHelper(wil::unique_cotaskmem_string const& appUserModelId, const winrt::guid& remoteId, ChannelDetails& channelInfo) noexcept try
```

Comment:
Just appId. AppUserModelId infers to the format PRAID!PFN which is specifically for packages with identity. This appId isn't specifically for those with identity right?